tensorflow/python/data/benchmarks/benchmark_base.py

```diff
@@ -124,3 +147,32 @@ def run_and_report_benchmark(self,
     extras["num_elements"] = num_elements
     self.report_benchmark(
         wall_time=wall_time, iters=iters, name=name, extras=extras)
+    return wall_time
+
+  def _consume_dataset(self, dataset, num_elements):
+    """Consume the `num_elements` of the dataset.
```

Comment:
"Consumes `num_elements` of the dataset."

Reply:
addressed in the commit